scripts/create-component/package-template/.storybook/config.js

```diff
@@ -0,0 +1,16 @@
+import addons from '@storybook/addons'
+import { addDecorator, configure } from '@storybook/react'
+
+import centerDecorator from '@pluralsight/ps-design-system-storybook-addon-center'
+import themeDecorator from '@pluralsight/ps-design-system-storybook-addon-theme'
+
+addDecorator(centerDecorator)
+addDecorator(themeDecorator(addons))
+
+const req = require.context('../stories', true, /\.story\.js$/)
```

Comment:
Is the *.story.js a new convention?  Looks like it's in the latest halo-touching stuff.  I think I'm ok with it.  Just didn't notice it before.

Reply:
Perhaps related to this question and resolvable later... perhaps we should move *.story.js files out of '/stories' and into '/.storybook'.  Not sure if we need two storybook dirs in every package.

Reply:
sounds good to me. I can get it in place with this pr for new packages and we can refactor old packages down the road

Reply:
I hit a snag. storybook doesn't like stories in the .storybook folder. I decided to instead move the stories next to the component in src. thoughts?

Reply:
hmmm, seems a little odd.  And it's not really src.  We make the exception for specs.  We could do the same for __stories__, I suppose.

what's your fav location for these stories?